modules/cluster-logging-deploy-cli.adoc

```diff
@@ -368,9 +376,11 @@ However, an unmanaged deployment does not receive updates until cluster logging
 <4> Specify the length of time that Elasticsearch should retain each log source. Enter an integer and a time designation: weeks(w), hours(h/H), minutes(m) and seconds(s). For example, `7d` for seven days. Logs older than the `maxAge` are deleted. You must specify a retention policy for each log source or the Elasticsearch indices will not be created for that source.
 <5> Specify the number of Elasticsearch nodes. See the note that follows this list.
 <6> Enter the name of an existing storage class for Elasticsearch storage. For best performance, specify a storage class that allocates block storage. If you do not specify a storage class, {product-title} deploys cluster logging with ephemeral storage only.
-<7> Settings for configuring Kibana. Using the CR, you can scale Kibana for redundancy and configure the CPU and memory for your Kibana pods. For more information, see *Configuring Kibana*.
-<8> Settings for configuring the Curator schedule. Curator is used to remove data that is in the Elasticsearch index format prior to {product-title} 4.5 and will be removed in a later release.
-<9> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits. For more information, see *Configuring Fluentd*.
+<7> Specify the CPU and memory requests for Elasticsearch as needed. The defaults are 16Gi` for the memory request and `1m` for the CPU request.
+<8> Specify the memory requests for the Elasticsearch proxy as needed.The defaults are `64Mi` for the memory request and `100m` for the CPU request.
```

Comment:
The defaults for ES proxy are different in different branches
master: 
```
	defaultESProxyMemoryLimit   = "256Mi"
	defaultESProxyMemoryRequest = "256Mi"
```
https://github.com/openshift/elasticsearch-operator/blob/master/internal/k8shandler/defaults.go#L21

release-4.5
```
	defaultESProxyMemoryLimit   = "256Mi"
	defaultESProxyMemoryRequest = "256Mi"
```
https://github.com/openshift/elasticsearch-operator/blob/release-4.6/pkg/k8shandler/defaults.go#L20

release-4.5
```
	defaultESProxyMemoryLimit   = "64Mi"
	defaultESProxyMemoryRequest = "64Mi"
```
https://github.com/openshift/elasticsearch-operator/blob/release-4.5/pkg/k8shandler/defaults.go#L20